### PR TITLE
remove appx packaging

### PR DESCRIPTION
**Pull request by TravisEz13**
Appx is not used and never worked properly so removing it

### CODE REVIEW

File: tools/packaging/packaging.psm1 [hunk 1]
```diff
@@ -19,7 +19,7 @@ function Start-PSPackage {
         [string]$Name = "powershell",
 
         # Ubuntu, CentOS, Fedora, OS X, and Windows packages are supported
-        [ValidateSet("deb", "osxpkg", "rpm", "msi", "appx", "zip", "AppImage")]
+        [ValidateSet("deb", "osxpkg", "rpm", "msi", "appx", "zip", "AppImage", "nupkg")]
```
Comment (SteveL-MSFT) [inline tools/packaging/packaging.psm1]: shouldn't you remove `appx` here?

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: tools/packaging/packaging.psm1 (hunk 1 above)
Comment (TravisEz13) [inline tools/packaging/packaging.psm1]: Fixed

File: tools/packaging/packaging.psm1 [hunk 2]
```diff
@@ -19,7 +19,7 @@ function Start-PSPackage {
         [string]$Name = "powershell",
 
         # Ubuntu, CentOS, Fedora, OS X, and Windows packages are supported
-        [ValidateSet("deb", "osxpkg", "rpm", "msi", "appx", "zip", "AppImage")]
+        [ValidateSet("deb", "osxpkg", "rpm", "msi", "zip", "AppImage", "nupkg")]
```
Comment (SteveL-MSFT) [inline tools/packaging/packaging.psm1]: Isn't `nupkg` support part of your other PR?

Comment (TravisEz13) [inline tools/packaging/packaging.psm1]: fixed but squashed history to avoid conflict with nuget change

**Comment by adityapatwardhan** [comment]
@TravisEz13 Can you add in the description why we are making this change.

**Review by SteveL-MSFT — APPROVED** [review]
LGTM

**Comment by TravisEz13** [comment]
@daxian-dbw  Thanks for resolving confilcts

**Pull request merged**